### PR TITLE
Fix ad image sizing issue

### PR DESCRIPTION
**Pull request by danehansen**

### CODE REVIEW

**Comment by danehansen** [comment]
:eyeglasses: @dwick

File: src/views/components/Listing.jsx [hunk 1]
```diff
@@ -26,14 +26,15 @@ class Listing extends BaseComponent {
...
     this.state = {
-      compact: _isCompact(props),
+      compact: compact,
       expanded: false,
       loaded: false,
       tallestHeight: 0,
       reported: false,
       hidden: false,
-      width: 0,
+      width: compact ? 80 : ((globals().winWidth || 300) - 10),
```
Comment (dwick) [inline src/views/components/Listing.jsx]: can you explain these numbers?  is `width` the image size or the window size?

**Comment by dwick** [comment]
:haircut: question before fish

**Comment by dwick** [comment]
:nail_care: question before fish

**Comment by danehansen** [comment]
:haircut: @dwick

**Comment by curioussavage** [comment]
55d99d743e9136252b319b508d4e2cecfe3c2fc8

**Pull request closed without merging**